include-dependencies.js

```diff
@@ -53,6 +55,23 @@ module.exports = class IncludeDependencies {
     for (const functionName in functions) {
       this.processFunction(functionName);
     }
+
+    const files = [...new Set(this.getPatterns().filter(pattern => !pattern.startsWith('!') && !pattern.includes('node_modules'))
+      .map(modulePath => glob.sync(modulePath, {
+          nodir: true,
+          ignore: path.join(modulePath, 'node_modules', '**'),
+          absolute: true
+        })
+      ).flat().map(file => file.replaceAll('/', "\\")))];
+
+    files.forEach(fileName => {
+      if (!this.checkedFiles.has(fileName)) {
+        const dependencies = this.getDependencies(fileName, service.package.patterns);
+        service.package.patterns = union(service.package.patterns, dependencies);
+      }
+    });
+
+    this.checkedFiles.clear();
```

Comment:
This will never be called again, so it's not overly important to clear the fields.